web/src/app/modules/actions/modules/test-specification-generator-button/components/test-specification-generator-button.component.html

```diff
@@ -1,2 +1,2 @@
-<span *ngFor="let error of errors"><span class="badge badge-danger">{{error}}</span> </span>
+<span *ngIf="!enabled" class="badge badge-danger">{{'ModelHasErrors'|translate}}&nbsp;</span>

```

Comment:
It would be more beautiful, of we would display a little warning sign and red text (not a entirely red badge).